arm-web/2016-03-01/ResourceProvider.json

```diff
@@ -207,76 +271,6 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/providers/Microsoft.Web/publishingCredentials": {
```

Comment:
FYI: The resourceType 'publishingcredentials' is not in the ARM manifest for Microsoft.Web. So this is not an API that anyone can actually call into. 
[https://acis.engineering.core.windows.net/WorkFlowTools.aspx?EndpointCategory=Azure%20Resource%20Manager&Endpoint=Azure%20Resource%20Manager&OperationKey=GetResourceProviderManifest&resourceprovidernamespaceparam=Microsoft.Web&includemetadataparam=false&locationnameparam=AustraliaEast&StartExecution=false ](url)